patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -2105,6 +2220,14 @@
  			else if (shieldRaised) {
  				bodyFrame.Y = bodyFrame.Height * 10;
  			}
+@@ -25178,6 +_,7 @@
+ 			isPerformingJump_Sail = false;
+ 			isPerformingJump_Unicorn = false;
+ 			isPerformingJump_Santank = false;
++			ModExtraJumpLoader.ResetAllVisualEffects(this);
+ 		}
```

Comment:
Below this, rocket boots currently check `canJumpAgain_Cloud` to know when they should trigger. You'll need to change that to support after cloud double jumps

Reply:
Judging by the wiki, the rocket boots only interact with the cloud, not other jumps: `If the Cloud in a Bottle is also equipped, its double-jump effect will engage first; pressing and holding the jump key afterward while still in midair will then activate the Rocket Boots.`

So idk if we should touch this behavior